src/main/java/uk/co/onsdigital/discovery/metadata/api/service/MetadataServiceImpl.java

```diff
@@ -112,4 +137,27 @@ private DimensionMetadata convertDimension(String dataSetId, Dimension dimension
 
         return result;
     }
+
+    private DimensionMetadata convertHierarchyToDimension(Hierarchy hierarchy) {
+        final DimensionMetadata dimension = new DimensionMetadata();
+        dimension.setId(hierarchy.getId());
+        dimension.setName(hierarchy.getName());
+        dimension.setType(hierarchy.getType());
+        dimension.setHierarchical(true);
+        dimension.setUrl(urlBuilder.hierarchy(hierarchy.getId()));
+
+        return dimension;
+    }
+
+    private List<DimensionOption> convertHierarchyEntries(Collection<HierarchyEntry> entries) {
+        return DimensionViewType.HIERARCHY.convertValues(entries.stream().map(this::valueFromHierarchyEntry).collect(toList()));
+    }
+
+    private DimensionValue valueFromHierarchyEntry(final HierarchyEntry entry) {
+        final DimensionValue value = new DimensionValue();
+        value.setId(UUID.randomUUID());
```

Comment:
Yes, this is a bit of a hack to re-use the code for rendering hierarchical dimensions, so I create fake dimension values for each hierarchy entry. I'll check if I can leave the IDs null or if there is a more elegant way to reuse this.